packs/data/feats.db/enhanced-psychopomp-familiar.json

```
@@ -11,7 +11,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your psychopomp's continued service in Pharasma's name brings it favor from the goddess. It might change appearance to look like a different, more powerful kind of psychopomp. You can select four familiar or master abilities each day, instead of two, but two must be from the Psychopomp Familiar feat or the following ability.</p>\n<ul>\n<li><strong>Augury</strong> Your familiar can glimpse the strands of fate to give you a cryptic clue regarding your future. Your familiar can cast <em>@Compendium[pf2e.spells-srd.Augury]{Augury}</em>once per day using your magical tradition and spell DC. You must be at least 8th level to select this ability.</li>\n</ul>"
+            "value": "<p>Your psychopomp's continued service in Pharasma's name brings it favor from the goddess. It might change appearance to look like a different, more powerful kind of psychopomp. You can select four familiar or master abilities each day, instead of two, but two must be from the Psychopomp Familiar feat or the following ability.</p>\n<ul>\n<li>@Compendium[pf2e.familiar-abilities.Augury]{Augury}</li>\n</ul>"
```

```suggestion
            "value": "<p>Your psychopomp's continued service in Pharasma's name brings it favor from the goddess. It might change appearance to look like a different, more powerful kind of psychopomp. You can select four familiar or master abilities each day, instead of two, but two must be from the Psychopomp Familiar feat or the following ability.</p>\n<ul>\n<li><strong>@Compendium[pf2e.familiar-abilities.Augury]{Augury}</strong> Your familiar can glimpse the strands of fate to give you a cryptic clue regarding your future. Your familiar can cast <em>@Compendium[pf2e.spells-srd.Augury]{Augury}</em>once per day using your magical tradition and spell DC. You must be at least 8th level to select this ability.</li>\n</ul>"
```

This one is optional